src/cursor/cur_backup.c

```diff
@@ -157,19 +202,23 @@ __wt_curbackup_open(WT_SESSION_IMPL *session, const char *uri, WT_CURSOR *other,
     *cursor = iface;
     cursor->session = (WT_SESSION *)session;
     cursor->key_format = "S";  /* Return the file names as the key. */
-    cursor->value_format = ""; /* No value. */
+    cursor->value_format = ""; /* No value, for now. */
 
     session->bkp_cursor = cb;
+    othercb = (WT_CURSOR_BACKUP *)other;
+    if (othercb != NULL)
+        WT_CURSOR_BACKUP_CHECK_STOP(othercb);
 
     /*
      * Start the backup and fill in the cursor's list. Acquire the schema lock, we need a consistent
      * view when creating a copy.
      */
     WT_WITH_CHECKPOINT_LOCK(
-      session, WT_WITH_SCHEMA_LOCK(session, ret = __backup_start(session, cb, other != NULL, cfg)));
+      session, WT_WITH_SCHEMA_LOCK(session, ret = __backup_start(session, cb, othercb, cfg)));
```

Comment:
Should we split `__backup_start()` into two parts, duplicate and "not duplicate" cursor? They really don't share anything, do they?

Reply:
We could. I'll take a run at that. It could be deferred until a later ticket.